### ISSUE TITLE
Alpine-linux on ios ISH app....deploy help needed

### ISSUE THREAD

**Issue by administrateur-Sexflixxx** [michmich112/get-actions-dir-path#1]
I try do it as its writen but wont download the apk and the npm apk is already installed but i cant get it to start.... i put some screenshots here! just note that im a real beginner so help please
![F6DD3ACB-DDA3-48D9-8FD8-E4AB42403391](https://user-images.githubusercontent.com/115735398/197700202-1463b37d-d06f-4c57-85f3-a5531131df13.png)
![B592F668-8E87-4657-BE41-EBEB9563F837](https://user-images.githubusercontent.com/115735398/197700207-29a35d70-55a9-498d-8a69-b4816a72b1f2.png)
![F62C2A00-7B00-4E5A-A060-AE94144FA0CB](https://user-images.githubusercontent.com/115735398/197700208-93456921-a271-42dc-be4b-1b4ee03e0103.png)
![5C8D7936-84A5-4CC7-AAAD-F39253BE3E11](https://user-images.githubusercontent.com/115735398/197700211-900c8380-9d2f-456e-b04e-da1d9feaf996.png)

**Comment by michmich112** [comment]
hey @administrateur-Sexflixxx 
Could you give me a bit more insight as to what you're attempting to do?
This github actions allows you to extract the current version of any project from any file/location.
From the screenshots you are having an issue installing `npm` using `apk` onto your alpine machine, is that correct?

**Comment by administrateur-Sexflixxx** [comment]
> hey @administrateur-Sexflixxx 
> 
> Could you give me a bit more insight as to what you're attempting to do?
> 
> This github actions allows you to extract the current version of any project from any file/location.
> 
> From the screenshots you are having an issue installing `npm` using `apk` onto your alpine machine, is that correct?

@michmich112 Yes npm installing no issue its after that (and maybe before,im real beginner 😨🙄 im sorry) maybe i missed a step in process or im not doing the right commands and settings îm getting lost within all those codes languages and syntax 😤😂

**Comment by administrateur-Sexflixxx** [comment]
> hey @administrateur-Sexflixxx 
> 
> Could you give me a bit more insight as to what you're attempting to do?
> 
> This github actions allows you to extract the current version of any project from any file/location.
> 
> From the screenshots you are having an issue installing `npm` using `apk` onto your alpine machine, is that correct?

So in my organization îm setting up a azure devops pipeline (test) and installed extensions on both devops and github too but i clearly miss a step of something between this and that lol im sending you a feedback of errors of the workflows i failed maybe will help?...

**Comment by michmich112** [comment]
@administrateur-Sexflixxx are you using it on a github action? if so can you share a public version of the workflow file? 
No worries at all, normally every runner comes with npm and node pre-installed so there shouldn't be any problem

**Comment by administrateur-Sexflixxx** [comment]
@michmich112 ![image](https://user-images.githubusercontent.com/115735398/197930618-45cc3cd1-029b-4847-86c0-341050ff91e1.jpeg)![image](https://user-images.githubusercontent.com/115735398/197930642-dcf4cc92-7912-45ec-be9a-4000c03dcb2a.jpeg)![image](https://user-images.githubusercontent.com/115735398/197930655-0e5427c8-5575-40c9-8744-d2a9a9b2de65.png)![image](https://user-images.githubusercontent.com/115735398/197930663-36d47699-8948-4ff2-ad46-f335027d7fc7.png)![image](https://user-images.githubusercontent.com/115735398/197930680-a2681c55-6948-40a8-bb2a-3b3a86492823.png)![image](https://user-images.githubusercontent.com/115735398/197930690-12024469-40d7-43a1-a25d-987567fb48d2.png)![image](https://user-images.githubusercontent.com/115735398/197930705-05f02333-02a2-404f-a575-3e889051406a.png)![image](https://user-images.githubusercontent.com/115735398/197930714-978b5b8f-07d1-48b6-af4b-363c5532d123.png)![image](https://user-images.githubusercontent.com/115735398/197930722-787c5067-a64c-45a7-944c-4d5a935ffb03.png)

**Comment by michmich112** [comment]
hi @administrateur-Sexflixxx 
As far as I can see, this issue is not linked to the use of this Action on the Github actions product.
Stack exchange (https://unix.stackexchange.com/)[https://unix.stackexchange.com/] might be your best bet for help on this.
If you have a repo and/or a github action workflow file you can share with me, I'll be more able to help you :)
